server/data/data-library/src/main/java/org/apache/james/dlp/eventsourcing/EventSourcingDLPConfigurationStore.java

```diff
@@ -81,7 +82,7 @@ public void clear(Domain domain) {
 
     @Override
     public Optional<DLPConfigurationItem> fetch(Domain domain, Id ruleId) {
-        return list(domain)
+        return StreamSupport.stream(list(domain).spliterator(), false)
```

Comment:
Iterators.toStream()

Reply:
I introduce an _Iterables_ class for that, which is just a simplification of the _StreamSupport_ class -> no tests.